units/provisioning_setup_host.go

```diff
@@ -246,6 +253,65 @@ func (j *setupHostJob) runHostSetup(ctx context.Context, targetHost *host.Host,
 	return nil
 }
 
+// fetchJasper places the Jasper CLI on the host via SSH.
+func (j *setupHostJob) fetchJasper(ctx context.Context) error {
+	d, err := distro.FindOne(distro.ById(j.host.Distro.Id))
+	if err != nil {
+		grip.Error(message.WrapError(j.host.SetUnprovisioned(), message.Fields{
+			"operation": "setting host unprovisioned",
+			"distro":    j.host.Distro.Id,
+			"job":       j.ID(),
+			"host":      j.host.Id,
+		}))
+		return errors.Wrapf(err, "Error finding distro for host %s", j.host.Id)
+	}
+	j.host.Distro = d
+
+	cloudHost, err := cloud.GetCloudHost(ctx, j.host, j.env.Settings())
+	if err != nil {
+		return errors.Wrapf(err, "failed to get cloud host for %s", j.host.Id)
+	}
+
+	sshOptions, err := cloudHost.GetSSHOptions()
+	if err != nil {
+		grip.Error(message.WrapError(j.host.SetUnprovisioned(), message.Fields{
+			"operation": "setting host unprovisioned",
+			"distro":    j.host.Distro.Id,
+			"job":       j.ID(),
+			"host":      j.host.Id,
+		}))
+		return errors.Wrapf(err, "error getting ssh options for host %v", j.host.Id)
+	}
+
+	if err := j.doFetchJasper(ctx, "/usr/local/bin", sshOptions); err != nil {
```

Comment:
"/usr/local/bin" should probably not in an argument, but instead part of `doFetchJasper`

Reply:
Done.